modules/highgui/include/opencv2/highgui.hpp

```diff
@@ -496,7 +524,7 @@ The function sets the maximum position of the specified trackbar in the specifie
 
 @note
 
-**[Qt Backend Only]** winname can be empty (or NULL) if the trackbar is attached to the control
+<b>[Qt Backend Only]</b> winname can be empty (or NULL) if the trackbar is attached to the control
```

Comment:
@sturkmen72, problem is in the way doxygen parses comments: it will remove all `*` at the beginning of the line.

Please replace `**` to `__`, it is better to avoid HTML tags where possible.